src/main/java/DukeWriteFile.java

```diff
@@ -0,0 +1,44 @@
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * Writes the current tasks in the lst to a text document
+ * whenever the task list changes.
+ */
+public class DukeWriteFile {
+
+	protected static String filePath;
+
+	public DukeWriteFile(String filePath) {
+
+		this.filePath = filePath;
+	}
+
+
+	/**
+	 * Writes the current task in the list to the text document.
+	 *
+	 * @param textToaAdd Current tasks in the list.
+	 * @throws IOException
+	 */
+	public static void writeToFile(String textToaAdd) throws IOException {
+		FileWriter fw = new FileWriter(filePath);
```

Comment:
Consider try-with-resources.

Reply:
Could you elaborate more on what you meant by try-with resources?